docs/farming-&-staking/farming/intro.mdx

```diff
@@ -145,14 +145,4 @@ Before you begin, make sure you have a wallet to receive rewards. We recommend u
 
 For detailed information on network configurations, including port forwarding requirements and firewall settings, please refer to our [Port Forwarding & Firewall](/farming/guides/port-config) guide.
 
-:::
-
-## Dependencies
-
-:::caution  Windows No Output Bug
```

Comment:
Nazar's advice was to leave it here + also put the same caution right at the start of the windows space acres installation guide. Do you mind reverting this change, and copying the same caution there?